frontend/src/css/personalPage.css

```diff
@@ -34,7 +34,7 @@
     margin-top: 750px;
 }
 
-.popup-addfriend {
+.popup-add {
     position: fixed;
     top: -200px;
     left: 80vh;
```

Comment:
I think we should make the ``ModalWindow``'s position fixed and center it on the viewport (so you don't need to add this extra class.